packages/server/modules/shared/authz.ts

```diff
@@ -0,0 +1,293 @@
+import { Scopes } from '@/modules/core/helpers/mainConstants'
+import { getRoles } from '@/modules/shared'
+import { getStream } from '@/modules/core/services/streams'
+
+import {
+  BaseError,
+  ForbiddenError as FE,
```

Comment:
this was probably already here before, but these shortened error names are really unintuitive. i'm guessing originally there were other errors with the same name imported here? there don't seem to be any more, tho, so maybe the aliases can be removed?

or you could do something like
```js
import * as Errors from '@/modules/shared/errors'
...
throw new Errors.ForbiddenError()
```

Reply:
yeah, it was some name duplication. I'm not a fan of * imports, but we can def drop the aliases